sysroot_creator/scripts/create_cc_sysroot.py

```diff
@@ -127,31 +128,53 @@ def __str__(self):
             self.base_image, self.network_mode, self.nocache)
 
 
-def setup_cc_root_dir(platform: Platform) -> Path:
+def setup_cc_root_dir(
+  platform: Platform,
+  sysroot_directory: str = None) -> Path:
     """Create cross-compile root directory."""
     if not isinstance(platform, Platform):
         raise TypeError('Argument `platform` must be of type Platform.')
-    logger.info('Creating workspace directory...')
-    cc_root = Path.cwd() / platform.__str__
+
+    logger.info('Creating cross-compile directory...')
+    cc_root = Path.cwd() if sysroot_directory is None else Path(sysroot_directory)
+    cc_root = cc_root / platform.__str__
```

Comment:
```suggestion
    cc_root = cc_root / str(platform)
```

Let's avoid accessing private attributes.